options/env.go

```diff
@@ -31,14 +31,31 @@ const (
 	EnvGithubRepository        = "GITHUB_REPOSITORY"
 	EnvGithubRef               = "GITHUB_REF"
 	EnvGithubWorkspace         = "GITHUB_WORKSPACE"
-	EnvGithubAuthToken         = "GITHUB_AUTH_TOKEN" //nolint:gosec
+	EnvGithubAuthToken         = "INPUT_REPO_TOKEN" //nolint:gosec
```

Comment:
```suggestion
	EnvGithubAuthToken         = "GITHUB_AUTH_TOKEN" //nolint:gosec
	EnvInputRepoToken          = "INPUT_REPO_TOKEN" //nolint:gosec
```

Include both values.
If `GITHUB_AUTH_TOKEN` is present, then it is canonical.
We should only care about `INPUT_REPO_TOKEN` when `GITHUB_AUTH_TOKEN` is unset or empty.

Same comment for the `Env` struct below.